cli/src/main.rs

```diff
@@ -111,7 +100,7 @@ async fn upload_manifest(manifest_path: PathBuf) -> Result<()> {
     table.add_row(row!["Wasm task name:", manifest.fq_name()]);
     table.add_row(row!["Version:", manifest.version()]);
 
-    let url = build_url("storage/manifests".to_string(), Some("1"));
+    let url = build_url("storage/manifests".to_string(), Some("1")).await;
```

Comment:
I was tripping over `build_url` earlier this week; thoughts on a little refactor along these lines? Not asking you to do it necessarily but I just want to run it by you.

```-fn build_url(path: String, version: Option<&str>) -> String {
+fn build_url(path: String) -> String {
+    build_url_with_version(path, Some("1"))
+}
+
+fn build_url_with_version(path: String, version: Option<&str>) -> String {

Also: should we make `path` be a `&str` instead?